.changeset/silly-colts-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
```

Comment:
I did the same here https://github.com/biomejs/biome/pull/6896 but merged into `next`. I think it's fine to have them shipped in patches, but we should agree how we should ship new APIs from core :)

Reply:
I exposed those functions because I thought they were more useful for the playground, where there are single files.

This doesn't mean that this new method is useless, because we could use it for the JavaScript APIs as well :)

Reply:
I'll rebase this to `next` and ship with the methods you added :)